### DISCUSSION TITLE
ADE 4.5  from Kobo

### DISCUSSION THREAD

**Discussion by AriaAis** [apprenticeharper/DeDRM_tools#1881] [Q&A] [answered]
Hey there ,
Quick Question of mine . I downloaded a book of Kobo and accidently downloaded it with ADE 4.5  with the ACSM-file, but I´ve read on the internet that Kobo doesnt actually use the protection for 4.5? But when i try to convert it , it still says its protected.

The file I have right now is a Epub file , but if the book actually still has the DeDRM for 4.5 do I need to rebuy the book so I can download it then with ADE 2.0.1 on a fresh Account, so I wont get the hardened DeDRM?

Does anybody mind looking at the Epub itself if its removable?
I would have to add a link because it seems the book cant be drag and dropped here

**Comment by ElleKayEm** [comment]
See if you can open it with ADE 2.0.1 authorized with the same Adobe ID.

**Comment by AriaAis** [comment]
I get an Error message which says something along the lines: 
" Activationsserver-Error
Code E_ADEPT_INTERNAL
Digital Editions has encountered an Error on the Activationsserver"
 
 I hope I translated that understadably since its German for me.
 But yeah seems like I cant open it on ADE 2.0.1

**Comment by AriaAis** [comment, marked as answer]
Oh yeah just as a follow up , if I cant , can I somehow tell the ACSM file that it shouldnt use ADE 4.5 to download by well , I dunno cleaning my registry or something like that ? 
Because if I download the file with  ADE4.5 I wont be able to download it with ADE2.0.1 right? Because the file somewhat is locked up

**Reply by ElleKayEm** [reply]
I'd remove ADE 4.5, de-authorize ADE 2.0.1, delete the registry key, re-authorize with your Adobe ID.  Then see if you can open the acsm file with ADE 2.0.1.  If that works, then DeDRM should work too.  Be careful with the registry.  See this link:  https://helpx.adobe.com/digital-editions/kb/activation-errors-adobe-digital-editions.html

**Reply by AriaAis** [reply]
Ive just found my own solution (which almost the same as yours but I didnt bother to do anything with the registtry) but I think yours would work aswell.
All I did is press ctrl+shift+D to de-authorize my account with ADE2.0.1  uninstall , restart pc , install and drag the ACSM file into ADE 2.0.1 and it just started the download and I successfully converted it to a PDF.

Only thing that is weird is the size of the Page
![Unbenannt](https://user-images.githubusercontent.com/92499984/137535186-b5804905-0194-4e54-b71c-a3f6a51d8476.png)
es
Any recommandation for Artbooks what type to use ? ( I mean  PDF FB2 AWZ3 ... )

**Reply by ElleKayEm** [reply]
I would have thought pdf, but I don't know if there are more specific settings you could use.  MobileRead is a good place to look for info/ask questions for non-DRM related ebook issues.
